llarp/config/config.cpp

```diff
@@ -531,6 +531,18 @@ namespace llarp
           m_ExitAuths.emplace(exit, auth);
         });
 
+    conf.defineOption<bool>(
+        "network",
+        "auto-routing",
+        ClientOnly,
+        Default{true},
+        Comment{
+            "enable / disable auto routing. When using an exit lokinet will add routes to "
+            "the OS to make traffic go over the network interface via lokinet.",
+            "enabled by default.",
```

Comment:
Suggested rewording:

"Enable / disable automatic route configuration. When this is enabled and an exit is used Lokinet will automatically configure operating system routes to route traffic through the exit node.  This is enabled by default, but can be disabled to perform advanced exit routing configuration manually."

Reply:
done